app/internal/event.py

```diff
@@ -84,3 +90,23 @@ def get_messages(session: Session,
         messages.append(f'Same event happened {weeks_diff} weeks before too. '
                         f'Want to create another one {weeks_diff} after too?')
     return messages
+
+
+def get_location_coordinates(
+        address: str,
+        timeout: float = LOCATION_TIMEOUT
+        ) -> NamedTuple:
+    """Return location coordinates and accurate
+    address of the specified location."""
+    Location = namedtuple('Location', 'latitude, longitude, location')
+    geolocator = Nominatim(user_agent="calendar", timeout=timeout)
```

Comment:
Does this line send a request to the internet? If it is - it's kinda problem, as we try to keep our servers async

Reply:
Converted my code to work async.

Reply:
Fortunately, this module does support work with async.
Use [`async with`](https://geopy.readthedocs.io/en/stable/#async-mode)